README.md
# Classic Commerce [![Build Status](https://travis-ci.com/ClassicPress-plugins/classic-commerce.svg?branch=master)](https://travis-ci.com/ClassicPress-plugins/classic-commerce)

Welcome to the Classic Commerce repository on GitHub. This is a fork of WooCommerce and is still in development.

Classic Commerce is a simple, powerful and independent e-commerce platform. Sell anything with ease.

You can browse the source, look at open issues and keep track of development here.

We recommend all developers follow the [Issues](https://github.com/ClassicPress-plugins/classic-commerce/issues) and [Pull Requests](https://github.com/ClassicPress-plugins/classic-commerce/pulls) for the latest development updates.

## Roadmap
You can follow up on the development cycle by reading our [roadmap](https://github.com/ClassicPress-plugins/classic-commerce/wiki/Plugin-Roadmap).

## FAQs
**Can I install WooCommerce together with Classic Commerce?**
Yes, you can have both WooCommerce and Classic Commerce *installed* as plugins. However, you can't have them both *activated*. You will need to deactivate one before activating the other.

**What are the differences between WooCommerce and Classic Commerce?**
Classic Commerce is a fork of WooCommerce with all the WooCommerce Services and Jetpack integrations removed. This means there will be no "nags" or "upsells" at any stage when you are using Classic Commerce. You won't find any plugins or products that are promoted or pushed in any way.

**Can I import my products and settings from WooCommerce to Classic Commerce?**
There is no need to import anything when changing from WooCommerce to Classic Commerce. They share all the same classes and hooks so your existing WooCommerce records will be recognised by Classic Commerce and immediately available.

**What version of WooCommerce is Classic Commerce based on?**
Classic Commerce is a fork of WooCommerce 3.5.3.

**What plugins work with Classic Commerce?**
All of the extensions compatible with WooCommerce 3.5.3 should still be usable, provided they do not rely on Jetpack or WooCommerce Services. However, we strongly recommend that you test and monitor any extensions or plugins that you may need to provide extra functionality.

**How do I switch over from using WooCommerce to using Classic Commerce? Is it safe to uninstall WooCommerce?**
To try out Classic Commerce you can manually install it as a plugin. Make sure you deactivate WooCommerce before activating Classic Commerce. If you want to uninstall WooCommerce completely you can do so safely, but we recommend you take a full backup of your site first. Note that deactivating and deleting WooCommerce *normally* only removes the plugin and its files; the settings, orders, products, pages, etc are retained in the database. However, there is a possibility that a setting in your site’s wp-config file may over-ride this, so it is worth checking to make sure the following condition has not been included anywhere in that file: `define( 'WC_REMOVE_ALL_DATA', true);`

## Documentation
* [Classic Commerce Documentation](https://classiccommerce.cc/docs/)
* [Classic Commerce Code Reference](https://classiccommerce.cc/docs/)
* [WooCommerce REST API Docs](https://woocommerce.github.io/woocommerce-rest-api-docs/)

## Reporting Security Issues
To disclose a security issue to our team. (security@classicpress.net)

## Support
This repository is most suitable for development but can also be used for support. We will add a support tag to your issue. However, this might take a little longer to get a response. You can also use the dedicated support area on the [ClassicPress community forum](https://forums.classicpress.net/tags/classic-commerce/).

## Contributing to Classic Commerce
If you have a patch or have stumbled upon an issue with Classic Commerce core, you can contribute this back to the code. Please read our [contributor guidelines](https://github.com/ClassicPress-plugins/classic-commerce/blob/master/.github/CONTRIBUTING.md) for more information about how you can do this.



CHANGELOG.txt
== Changelog ==

= 1.0.5 - 2021-09-01
Fixed: Fix new section in payments tab not working (#332)

= 1.0.4 - 2021-07-18
Fixed: Fix uncorrect use of `$wpdb->esc_like` #328

= 1.0.3 - 2020-11-19
Fixed: Fix for failing travis builds #294
Fixed: Backport of WC security fix [e711a44](https://github.com/woocommerce/woocommerce/commit/e711a447feaf3d6020204e4319fdd5ba47c3f0b9). See [this post](See changelog archive for all versions](https://developer.woocommerce.com/2020/11/05/woocommerce-4-6-2-fix-release/). #293
Fixed: Update email template override to use classic-commerce directory instead of woocommerce #292

= 1.0.2 - No release

= 1.0.1 - 2020-10-17
Fixed: CC icon not showing in some installations #289

= 1.0.0 - 2020-10-01
New: Complete new set of sample products created specifically for CC #284
Updated: All documentation links now point to classiccommerce.cc #274, #278, #281
Fixed: CC icon wasn't turning blue on hover #265
Updated: Update manager updated to v2 #264
Updated: node-sass and handlebars dependencies #258, #259
Improved: Stock levels are now hidden on the dashboard widget if inventory is disabled #261
Improved: Additional checks made during build process #269
Tweaks: Some well-hidden instances of WC purple changes to CC aqua #270, #271

= 1.0.0-rc.2 - 2020-08-29
Fixed: Corrected color of two images to CC aqua, missed in rc1. #255

= 1.0.0-rc.1 - 2020-08-29
Fixed: Incorrect escaping of strings in setup wizard and order edit page. #244
Improved: Better checks and improved messaging when invalid file type chosen for CSV tax import. #245
Improved: CSS tweaks to enhance CC branding. New CC aqua color introduced. #247
Improved: Big improvements to CC build process. #250, #251
Tweaked: Removed all references to old ClassicPress-Research repository. #249

= 1.0.0-beta.1 - 2020-05-16
Improved automated testing on GitHub (#174)
Coding standards improvements (#174)
Added user agent (#202)
Various CSS tweaks
Various spacing and capitalization tweaks
Rename logo and icon image files as per CP naming standard (#210)
Updates to CodePotent Update Manager (#215)
Comb out remnant css after Install wizard + Jetpack + woo services removal (#204)
Backport security fixes from WC 3.8.0 and below (#218, #219, #220, #221, #222, #225, #228, #229, #230)
Improved compatibility with PHP 7.4, ready for upcoming ClassicPress 1.2.0 (#225)
Changed composer.json "type" back to "wordpress-plugin" for greater compatibility (#239, #240)

= 1.0.0-alpha.3 - 2020-01-20
NEW: Add usage tracking notice & opt-out
NEW: Change template path to look for classic-commerce in theme folder
UPDATE: Improve CC/Woo activation logic
UPDATE: Change compatibility plugin warning message
FIX: Fix deactivating WC/CC issue and change notice
FIX: Prevent PHP fatal error when trying to activate WC while CC is still active
FIX: Fix for CC admin menu icon display issues
FiX: Prevent the compatibility plugin notice from appearing when it shouldn't
Other minor tweaks

= 1.0.0-alpha.2 - 2019-12-29
Fix for issues #159 and #162 - notices displaying when they shouldn't

= 1.0.0-alpha1 - 2019-12-28
First alpha release of Classic Commerce

= 0.1.0 - 2018-05-24 =
WIP - Forking and documenting Classic Commerce 0.1.0

= WooCommerce 3.4.5 - 2018-08-29 =
[See changelog archive for all versions](https://raw.githubusercontent.com/ClassicPress-plugins/classic-commerce/master/CHANGELOG-Archive.txt).


i18n/states/BD.php
<?php
/**
 * Bangladeshi states (districts)
 *
 * @package ClassicCommerce/i18n
 * @version WC-3.5.3
 */

global $states;

defined( 'ABSPATH' ) || exit;

$states['BD'] = array(
	'BD-05' => __( 'Bagerhat', 'classic-commerce' ),
	'BD-01' => __( 'Bandarban', 'classic-commerce' ),
	'BD-02' => __( 'Barguna', 'classic-commerce' ),
	'BD-06' => __( 'Barishal', 'classic-commerce' ),
	'BD-07' => __( 'Bhola', 'classic-commerce' ),
	'BD-03' => __( 'Bogura', 'classic-commerce' ),
	'BD-04' => __( 'Brahmanbaria', 'classic-commerce' ),
	'BD-09' => __( 'Chandpur', 'classic-commerce' ),
	'BD-10' => __( 'Chattogram', 'classic-commerce' ),
	'BD-12' => __( 'Chuadanga', 'classic-commerce' ),
	'BD-11' => __( "Cox's Bazar", 'classic-commerce' ),
	'BD-08' => __( 'Cumilla', 'classic-commerce' ),
	'BD-13' => __( 'Dhaka', 'classic-commerce' ),
	'BD-14' => __( 'Dinajpur', 'classic-commerce' ),
	'BD-15' => __( 'Faridpur ', 'classic-commerce' ),
	'BD-16' => __( 'Feni', 'classic-commerce' ),
	'BD-19' => __( 'Gaibandha', 'classic-commerce' ),
	'BD-18' => __( 'Gazipur', 'classic-commerce' ),
	'BD-17' => __( 'Gopalganj', 'classic-commerce' ),
	'BD-20' => __( 'Habiganj', 'classic-commerce' ),
	'BD-21' => __( 'Jamalpur', 'classic-commerce' ),
	'BD-22' => __( 'Jashore', 'classic-commerce' ),
	'BD-25' => __( 'Jhalokati', 'classic-commerce' ),
	'BD-23' => __( 'Jhenaidah', 'classic-commerce' ),
	'BD-24' => __( 'Joypurhat', 'classic-commerce' ),
	'BD-29' => __( 'Khagrachhari', 'classic-commerce' ),
	'BD-27' => __( 'Khulna', 'classic-commerce' ),
	'BD-26' => __( 'Kishoreganj', 'classic-commerce' ),
	'BD-28' => __( 'Kurigram', 'classic-commerce' ),
	'BD-30' => __( 'Kushtia', 'classic-commerce' ),
	'BD-31' => __( 'Lakshmipur', 'classic-commerce' ),
	'BD-32' => __( 'Lalmonirhat', 'classic-commerce' ),
	'BD-36' => __( 'Madaripur', 'classic-commerce' ),
	'BD-37' => __( 'Magura', 'classic-commerce' ),
	'BD-33' => __( 'Manikganj ', 'classic-commerce' ),
	'BD-39' => __( 'Meherpur', 'classic-commerce' ),
	'BD-38' => __( 'Moulvibazar', 'classic-commerce' ),
	'BD-35' => __( 'Munshiganj', 'classic-commerce' ),
	'BD-34' => __( 'Mymensingh', 'classic-commerce' ),
	'BD-48' => __( 'Naogaon', 'classic-commerce' ),
	'BD-43' => __( 'Narail', 'classic-commerce' ),
	'BD-40' => __( 'Narayanganj', 'classic-commerce' ),
	'BD-42' => __( 'Narsingdi', 'classic-commerce' ),
	'BD-44' => __( 'Natore', 'classic-commerce' ),
	'BD-45' => __( 'Nawabganj', 'classic-commerce' ),
	'BD-41' => __( 'Netrakona', 'classic-commerce' ),
	'BD-46' => __( 'Nilphamari', 'classic-commerce' ),
	'BD-47' => __( 'Noakhali', 'classic-commerce' ),
	'BD-49' => __( 'Pabna', 'classic-commerce' ),
	'BD-52' => __( 'Panchagarh', 'classic-commerce' ),
	'BD-51' => __( 'Patuakhali', 'classic-commerce' ),
	'BD-50' => __( 'Pirojpur', 'classic-commerce' ),
	'BD-53' => __( 'Rajbari', 'classic-commerce' ),
	'BD-54' => __( 'Rajshahi', 'classic-commerce' ),
	'BD-56' => __( 'Rangamati', 'classic-commerce' ),
	'BD-55' => __( 'Rangpur', 'classic-commerce' ),
	'BD-58' => __( 'Satkhira', 'classic-commerce' ),
	'BD-62' => __( 'Shariatpur', 'classic-commerce' ),
	'BD-57' => __( 'Sherpur', 'classic-commerce' ),
	'BD-59' => __( 'Sirajganj', 'classic-commerce' ),
	'BD-61' => __( 'Sunamganj', 'classic-commerce' ),
	'BD-60' => __( 'Sylhet', 'classic-commerce' ),
	'BD-63' => __( 'Tangail', 'classic-commerce' ),
	'BD-64' => __( 'Thakurgaon', 'classic-commerce' ),
);
